libs/blocks/gnav/gnav.js

```diff
@@ -421,14 +421,29 @@ class Gnav {
   };
 
   decorateSignIn = (blockEl, profileEl) => {
+    const profileDropDown = blockEl.querySelector(':scope > div:nth-child(2)');
```

Comment:
Is there a more specific selector available? It seems like relying on the order could make this fragile if the authoring isn't done exactly right

Reply:
Not with out writing a new one, based on this second child option.